packages/eslint-plugin-expo/CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+## Creating a new rule
+
+To create a new rules, add the following files:
+- `src/rules/newRule.ts`
+- `src/__tests__/newRule.ts`
+- `docs/rules/newRule.md`
```

Comment:
```suggestion
To create a new rules, add the rule definition, a test, and documentation:

```bash
touch src/rules/newRule.ts
touch src/__tests__/newRule.ts
touch docs/rules/newRule.md
```
```

slight preference for something you can copy and paste. we could condense into a one-liner by just passing multiple files into `touch` but i think this format also makes it easy to copy and paste individual lines if you want